test/e2e/e2e-test/src/test/resources/expected/als/endpoints-productpage.yml

```diff
@@ -15,4 +15,4 @@
 
 endpoints:
   - key: not null
-    label: /productpage
+    label: GET:/productpage
```

Comment:
When do we change this? If this pull request? I remember we had this before.

Reply:
> When do we change this? If this pull request? I remember we had this before.

No. In this PR, not before